src/utils/post_files_to_qa.py

```diff
@@ -8,8 +8,7 @@
 }
 
 response_one = requests.get(
-    'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=' +
-    resource,
+    'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=' + resource,
```

Comment:
This file either need to be updated or removed. Since the authentication for sandbox and qa changed